app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -76,32 +68,77 @@ const useStyles = makeStyles((theme) => ({
     justifyContent: "space-evenly",
     padding: 0,
   },
+  drawerPaper: {
+    padding: theme.spacing(2),
+    width: drawerWidth,
+  },
+  drawerHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  drawerTitle: {
+    alignSelf: "center",
+    fontWeight: "bold",
+    paddingLeft: theme.spacing(1),
+  },
   gutters: {
     [theme.breakpoints.up("md")]: {
       paddingLeft: theme.spacing(3),
       paddingRight: theme.spacing(3),
     },
+    justifyContent: "space-between",
     paddingLeft: 0,
     paddingRight: 0,
   },
-  search: {
-    [theme.breakpoints.down("sm")]: {
-      marginInlineEnd: theme.spacing(1),
-    },
+  nav: {
+    display: "flex",
+  },
+  icon: {
+    marginLeft: theme.spacing(1),
+  },
+  bug: {
     alignItems: "center",
     display: "flex",
-    flexGrow: 3,
     justifyContent: "flex-end",
+    [theme.breakpoints.down("md")]: {
+      paddingRight: theme.spacing(2),
+    },
   },
   title: {
+    alignSelf: "center",
     fontWeight: "bold",
   },
 }));
 
 export default function Navigation() {
   const classes = useStyles();
-
   const authenticated = useAuthContext().authState.authenticated;
+  const [open, setOpen] = React.useState(false);
+
+  const logoutButton = <NavButton route={logoutRoute} label={LOG_OUT} />;
+
+  const drawer = (
```

Comment:
```suggestion
  const drawerItems = (
```
Since it's not the whole drawer